aspnetcore/blazor/get-started.md

```diff
@@ -15,91 +15,115 @@ By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.
+   * Install the [.NET Core 3.1.201 or later (Preview) SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1).
+   * Run the following command in a command shell. The [Microsoft.AspNetCore.Components.WebAssembly.Templates](https://www.nuget.org/packages/Microsoft.AspNetCore.Components.WebAssembly.Templates/) package has a preview version while Blazor WebAssembly is in preview.
+
+   ```dotnetcli
+   dotnet new -i Microsoft.AspNetCore.Components.WebAssembly.Templates::3.2.0-preview3.20168.3
+   ```
 
    > [!NOTE]
-   > If you installed the Blazor Visual Studio extension for a prior preview release of ASP.NET Core Blazor (Preview 6 or earlier), you can uninstall the extension. Installing the Blazor templates in a command shell is now sufficient to surface the templates in Visual Studio.
+   > .NET Core SDK version 3.1.201 or later is **required** to use the 3.2 Preview 3 Blazor WebAssembly template. Confirm the installed .NET Core SDK version by running `dotnet --version` in a command shell.
 
-   # [Visual Studio Code](#tab/visual-studio-code)
+1. Install [Visual Studio Code](https://code.visualstudio.com/).
 
-   1\. Install [Visual Studio Code](https://code.visualstudio.com/).
+1. Install the latest [C# for Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp).
 
-   2\. Install the latest [C# for Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp).
+1. For a Blazor WebAssembly experience, execute the following command in a command shell:
 
-   3\. For a Blazor WebAssembly experience, execute the following command in a command shell:
+   ```dotnetcli
+   dotnet new blazorwasm -o WebApplication1
+   ```
 
-      ```dotnetcli
-      dotnet new blazorwasm -o WebApplication1
-      ```
+   For a Blazor Server experience, execute the following command in a command shell:
 
-      For a Blazor Server experience, execute the following command in a command shell:
+   ```dotnetcli
+   dotnet new blazorserver -o WebApplication1
+   ```
 
-      ```dotnetcli
-      dotnet new blazorserver -o WebApplication1
-      ```
+   For information on the two Blazor hosting models, *Blazor Server* and *Blazor WebAssembly*, see <xref:blazor/hosting-models>.
 
-      For information on the two Blazor hosting models, *Blazor Server* and *Blazor WebAssembly*, see <xref:blazor/hosting-models>.
+1. Open the *WebApplication1* folder in Visual Studio Code.
 
-   4\. Open the *WebApplication1* folder in Visual Studio Code.
+1. For a Blazor Server project, the IDE requests that you add assets to build and debug the project. Select **Yes**.
 
-   5\. For a Blazor Server project, the IDE requests that you add assets to build and debug the project. Select **Yes**.
+1. If using a Blazor Server app, run the app using the Visual Studio Code debugger. If using a Blazor WebAssembly app, execute `dotnet run` from the app's project folder.
```

Comment:
Running the app using the VS Code debugger should also work for Blazor WebAssembly apps now too. There's an extra step when launching the browser if the user actually wants to debug something, but since the point here is just to get the app running, I think we can skip that in these steps.